src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -61,11 +61,14 @@ sealed trait Task {
 
   def runSpecId: PathId = taskId.runSpecId
 
+  def status: Task.Status
```

Comment:
I think we should refactor/remove the `def launched: Option[Launched]` usage in the follow up issue https://github.com/mesosphere/marathon/issues/4153